### PR TITLE
Change ARM template for testing infrastructure to forward events to topic directly

### LINKED ISSUE #114: Change ARM template for testing infrastructure to forward events to topic directly
Change ARM template for testing infrastructure to forward events to topic directly rather than via a Logic App.

### PR DESCRIPTION
**Pull request by tomkerkhove**
Fixes #114

### CODE REVIEW

**Comment by stijnmoreels** [comment]
You probably have already triggered the build for this?

File: docs/media/schematics/schematics.drawio [hunk 1]
```diff
@@ -0,0 +1 @@
...MC9NnmvIDATOOB8ynMVesN2xZJlE0pxFluS/Lh/JX2FPO6E/cqHHyH+WhYS9+zgOF04LC6oEC9EABh0Jidpyu2BP6VUXKeEgDGqPorrZ+0Aldt3mgNFGo/IM53ylYUMbpYcxENNnuWfnLCz9k4QaWxcW2WbnYqVIxdjngw/iI+dGMufhAYCyVGhELMD/QzunHm+EIcbLWx3F29KwOevcxxwETr6axqPiGh1aT75uu26cmz1k60BlGNbNrq8buyV+C6r9b+qoCfDUg4GsEIObP9aDrwYxpjFuRVyYUkSAWxQj70oOMJRE7m1tlXhHPyxNiH8Z6kuzALMtqiMYZkGqv+eDENd8cCimng9TdWs7FBJ/kvtV0IhnRJRNPAc/ZWlg8sm6b0gTFpe0bTYhbVoiBNesaZs1Liw5iG5nIx5RjiVOCGRFTxkyZHuvyUe2SLS533IYGeZ+ySzK5Igy5905uxc6e3DqZLQE4j6QtB+pMmcAOU6odY5Mp06GYMukwRYHcljEjKA726q2By2k4KFEzyovsUQg1pIz8Ej1Q6QwxV/WGoCCjS+LgIe+8MEAPjlMX9+O4nEIbnglHOANHcZxeEsbyvPnbbS2Pbhknb9wyqq6PlOT5s8z409bibEDdRbHlVb2ax9qWI2umZwTBLd1RMcGOo5wl1XzeQJzuEfoxy7upFaNFIjdj60rcOPZu5SWFzKsRSlPiHqHDlvCKDeK5IMMEqmJNBlnYNZnxXL2x4tDkgueTyYlksw5mjTG4MYGlwT22Z53EAQ/s6d5KW521Vjsfncpaw2g5slqO9rBWsAXtGs0S2SA9MF6r9R6oXRKJh8Jjq3fpnvp+ioeRTfcwOGcYcSxsT9kydRlJil3xEPoxNP2cKJ9G1v2h6aVfPR5Kw2qgZ5SS+ba8XUvJgLata8l0LqklB7SWAPuVS4A5aTmyJjenLQIvlRM09Tc570VN3SP+Akf4f6Qm40pqss+lJvlpQ+OGbVxSTG0NGM4rxeS0ljgTDCOl9gbQBu9FS91bmlvmZmIq4IEsGVKHjVd/Innxwfqqd2iG3Q/T0QO3PdhJrXvi/opRNP5OWdRzM3Mf+wyJ0GUuzxi+MHQXuRNpS/8dQDTtQCSvNkkcCONVAbnI5YYN3h0g3Q9nRz62DKKGS3x9gdblgi+K9ff7Yv2p/wvCuvsP</diagram></mxfile>
```
Comment (stijnmoreels) [inline docs/media/schematics/schematics.drawio]: Not sure what this is?

**Comment by tomkerkhove** [comment]
Trigger build for what? :)

This is just the ARM template we provide for folks to use to get started, not perse what we use internally.

File: docs/media/schematics/schematics.drawio (hunk 1 above)
Comment (tomkerkhove) [reply docs/media/schematics/schematics.drawio]: This is the schematic file used to generate the image

**Pull request merged**